jobs/config.json

```diff
@@ -8849,6 +8849,15 @@
       "UNKNOWN"
     ]
   },
+  "periodic-ci-kubernetes-e2e-kubeadm-pinned-releases": {
```

Comment:
nit on the naming. This actually verifies that the debs are pushed as they should and is therefore not kubeadm-specific.
I'd rather call this something more generic like `periodic-kubernetes-e2e-debs-pushed` or similar

@krzyzacy WDYT?